packages/terra-alert/src/Alert.jsx

```diff
@@ -164,12 +164,10 @@ const Alert = ({
   }
 
   const alertMessageContent = (
-    <div className={alertSectionClassName}>
-      <strong className={cx('title')}>{title || defaultTitle}</strong>
-      <div>
-        {children}
-      </div>
-    </div>
+    <p className={alertSectionClassName}>
```

Comment:
just curious, what's the difference between a div and paragraph for screen readers?

Reply:
It's more semantic. It reads it as text as opposed to a generic container, and I'm finding that it reads better when you have a variety of spans and other nodes for text decorating.